CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 
 * [#8146](https://github.com/rubocop-hq/rubocop/pull/8146): Use UTC in RuboCop todo file generation. ([@mauro-oto][])
 * [#8149](https://github.com/rubocop-hq/rubocop/pull/8149): Cop "Metrics/CyclotomicComplexicy" now counts `&.`, `||=`, `&&=` and blocks known to iterate. Default bumped from 6 to 7. ([@marcandre][])
```

Comment:
There's a spelling error from another commit here. Could you fix it in this PR? `CyclotomicComplexicy`???

Reply:
Oops, fixed.